src/EnergyPlus/MixedAir.cc

```diff
@@ -4681,6 +4681,15 @@ void CalcOAMixer(EnergyPlusData &state, int const OAMixerNum)
     // Mixed air temperature is calculated from the mixed air enthalpy and humidity ratio.
     state.dataMixedAir->OAMixer(OAMixerNum).MixTemp =
         Psychrometrics::PsyTdbFnHW(state.dataMixedAir->OAMixer(OAMixerNum).MixEnthalpy, state.dataMixedAir->OAMixer(OAMixerNum).MixHumRat);
+
+    // Check for saturation temperature > dry-bulb temperature and modify temperature at constant enthalpy
+    Real64 T_sat =
+        Psychrometrics::PsyTsatFnHPb(state, state.dataMixedAir->OAMixer(OAMixerNum).MixEnthalpy, state.dataMixedAir->OAMixer(OAMixerNum).MixPressure);
+    if (state.dataMixedAir->OAMixer(OAMixerNum).MixTemp < T_sat) {
+        state.dataMixedAir->OAMixer(OAMixerNum).MixTemp = T_sat;
+        state.dataMixedAir->OAMixer(OAMixerNum).MixHumRat =
+            Psychrometrics::PsyWFnTdbH(state, T_sat, state.dataMixedAir->OAMixer(OAMixerNum).MixEnthalpy);
+    }
```

Comment:
Pretty minimal fix here!  My only suggestion would have been to `auto & mixer = state.dataMixedAir->OAMixer(OAMixerNum);` so that the code would be reduced to:

```c++
Real64 T_sat = Psychrometrics::PsyTsatFnHPb(state, mixer.MixEnthalpy, mixer.MixPressure);
if (mixer.MixTemp < T_sat) {
    mixer.MixTemp = T_sat;
    mixer.MixHumRat = Psychrometrics::PsyWFnTdbH(state, T_sat, mixer.MixEnthalpy);
}
```

which is much more satisfying :)   But not necessary.  So just a note.